AUTHORS

```diff
@@ -26,3 +26,4 @@ Richard Eklycke <richard@eklycke.se>
 Sanil Raut <sr1990003@gmail.com>
 Sergio Ammirata <sergio@ammirata.net>
 The Chromium Authors <*@chromium.org>
+Ivi.ru LLC <*@ivi.ru>
```

Comment:
Can you move it to line 20, i.e. keep it sorted in alphabetical order

Reply:
Da